Intersections_3/include/CGAL/Intersections_3/internal/tetrahedron_intersection_helpers.h

```diff
@@ -82,52 +81,52 @@ void fill_segments_infos(std::vector<Segment>& segments,
     wrapped_segments.push_back(Wrapped_segment(s));
 
   std::vector<Segment> bis = segments;
-  for(int plouf = 0; plouf < bis.size()-1; ++plouf)
+  for(int dummy = 0; dummy < bis.size()-1; ++dummy)
```

Comment:
This loop looks very wrong, because at the same time `dummy` is incremented, `bis` is also modified by a `pop_back()`.

Why not the following?
```c++
while(!bis.empty())
{
```

Reply:
Indeed... I must have been tired